actions/class.Runner.php

```diff
@@ -341,7 +341,7 @@ public function submitItemState()
         $serviceCallId = $this->getRequestParameter('testServiceCallId');
 
         try {
-            $serviceContext = $this->getServiceContext(false);
+            $serviceContext = $this->getServiceContext(true);
```

Comment:
Why state should be stored when session is paused?

As far as i understand this is expectable behavior. Data must not be stored anymore if session is paused/terminated.

Without this fix test taker gets an error after submitting item (and GUI no longer works)  if proctor terminates session.